ansible/configs/kni-osp/env_vars.yml

```diff
@@ -0,0 +1,73 @@
+# sample vars configuration file
+#
+# This file is passed to ansible-playbook to set key vars which need to be set
+# and typically customized for a sucessful deployment.
+#
+# Usage: ansible-playbook main.yml -e @.../sample_vars.yml
+#
+# Ideally make and keep a copy OUTSIDE your repo, especially if using Cloud Credentials
+# Credentials should be set seperately i.e. ~/secrets.yml and passed in with
+# a 2nd `-e` argument i.e. -e ~/secrets.yml
+
+#CHANGEME
+guid: kniosp                            # Your Global UNIQUE Identifier
+
+env_type: kni-osp          # Name of config to deploy
+cloud_provider: osp                     # Which AgnosticD Cloud Provider to use
+
+output_dir: /tmp/output_dir                # Writable working scratch directory
+email: name@example.com                   # User info for notifications
+
+#repo_method: file
+repo_method: satellite
+
+set_repositories_satellite_hostname: labsat.opentlc.com
+set_repositories_satellite_activationkey: 'TBD'
+
+install_student_user: true
+#CHANGEME or put in the secret
+#own_repo_path: http://admin.example.com/repos/version
+
+# Cloud specfic settings - example given here for OSP
+
+# The domain that you want to add DNS entries to
+osp_cluster_dns_zone: students.osp.opentlc.com
+
+# The dynamic DNS server you will add entries to.
+# NOTE: This is only applicable when {{ use_dynamic_dns}} is true
+osp_cluster_dns_server: ddns01.opentlc.com
+
+___image: rhel-8.1  # red
+___bmc_image: rhel-8.1-ospbmc # red
+
+bmc_image_file: rhel-8.1-ospbmc
+#bmc_image_file: rhel-server-7.7-update-2
+bastion_image_file: rhel-8.1
+
+worker_image_file: rhel-8.1
+worker_instance_count: 3
+
+master_image_file: rhel-8.1
+master_instance_count: 3
+
+provision_image_file: rhel-8.1
+update_all_packages: false
+
+openshift_fip_provision: true
+
+ipmi_image: pxeboot
+
+my_ocp_domain: students.osp.opentlc.com
+
+my_static_pull: '{ "auths": { "provision.{{ my_ocp_domain }}:5000": {"auth": "ZHVtbXk6ZHVtbXk=","email": "{{ email }}"} } }'
+
+my_pull_secret: CHANGEME
```

Comment:
This var needs to be redefined in the agnosticv config to reference the secret var.

Reply:
You can reference `ocp4_token`